docs/api-reference/aggregation-layers/aggregation-layer.md

```diff
@@ -1,54 +1,38 @@
-# AggregationLayer
+# AggregationLayer (Experimental)
 
-All of the layers in `@deck.gl/aggregation-layers` module perform some sort of data aggregation. All these layers perform aggregation with different parameters (CPU vs GPU, aggregation to rectangular bins vs hexagon bins, world space vs screen space, aggregation of single weight vs multiple weights etc).
+`AggregationLayer` is the base class for all layers in `@deck.gl/aggregation-layers` module. It implements the most common tasks for aggregation with flexibility of customizations. 
 
-`AggregationLayer` and `GridAggregationLayer` perform most of the common tasks for aggregation with flexibility of customizations. This document describes what `AggregationLayer` does and how to use it in other aggregation layers.
+`AggregationLayer` extends [CompositeLayer](../core/composite-layer.md).
 
+## Methods
 
-`AggregationLayer` is subclassed form `CompositeLayer` and all layers in `@deck.gl/aggregation-layers` are subclassed from this Layer.
+Any layer subclassing the `AggregationLayer` must implement the following methods:
 
-## Integration with `AttributeManager`
+#### `getAggregatorType` {#getaggregatortype}
 
-This layer creates `AttributeManager` and makes it available for its subclasses. Any aggregation layer can add attributes to the `AttributeManager` and retrieve them using `getAttributes` method. This enables using `AttributeManager`'s features and optimization for using attributes. Also manual iteration of `data` prop can be removed and attributes can be directly set on GPU aggregation models or accessed directly for CPU aggregation.
+Returns a string that indicates the type of aggregator that this layer uses, for example `'gpu'`. This method is invoked with each `updateState` lifecycle. If the type string does not match its previous value, the existing aggregator will be disposed and recreated.
```

Comment:
Can this be reworded somehow, the words type and method are a little confusing...

```suggestion
Returns a string that indicates the type of aggregator that this layer uses, for example `'gpu'`. This aggregator is invoked with each `updateState` lifecycle. If the type string does not match its previous value, the existing aggregator will be disposed and recreated.
```

Reply:
The phrase "this method" refers to `getAggregatorType()`, so it doesn't make sense to change it to "this aggregator". This sentence is clearly not explaining the process well, but I don't know how to improve it.

Reply:
> The phrase "this method" refers to `getAggregatorType()`, so it doesn't make sense to change it to "this aggregator". This sentence is clearly not explaining the process well, but I don't know how to improve it.

Sounds good, just thinking:
- Maybe align the wording `getAggregationMethod()` then talking about "this method" makes a bit more sense. 
- Maybe the term "lifecycle" here may not be that helpful? 
- Or how about: `the aggregation selected by 'getAggregationMethod()' is run during each 'updateState' call`.
- Or: `the selected aggregation is run during each 'updateState' call`.